lib/synth-audio.js

```diff
@@ -585,21 +585,25 @@ const synthCustomVendor = async(logger, {credentials, stats, language, voice, te
   }
 };
 
-const synthElevenlabs = async(logger, {credentials, stats, language, voice, text}) => {
-  const {api_key, model_id} = credentials;
+const synthElevenlabs = async(logger, {credentials, options, stats, language, voice, text}) => {
+  const {api_key, model_id, options: credOpts} = credentials;
+  const opts = !!options && Object.keys(options).length !== 0 ? options : JSON.parse(credOpts || '{}');
+  const optimize_streaming_latency = opts.optimize_streaming_latency ?
+    `?optimize_streaming_latency=${opts.optimize_streaming_latency}` : '';
   try {
     const post = bent('https://api.elevenlabs.io', 'POST', 'buffer', {
       'xi-api-key': api_key,
       'Accept': 'audio/mpeg',
       'Content-Type': 'application/json'
     });
-    const mp3 = await post(`/v1/text-to-speech/${voice}`, {
+    const mp3 = await post(`/v1/text-to-speech/${voice}${optimize_streaming_latency}`, {
       text,
       model_id,
       voice_settings: {
```

Comment:
sjhouldnt we look to see if `options.voice_settings` have been provided and use those if so?